.github/workflows/dependencies_hip.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 The AMReX Community
+#
+# License: BSD-3-Clause-LBNL
+# Authors: Axel Huebl
+
+# search recursive inside a folder if a file contains tabs
+#
+# @result 0 if no files are found, else 1
+#
+
```

Comment:
I copied this from an old script to transform non-ASCII characters in CI at some point, ooopsi :D
```suggestion
```